Classes/Hooks/KeSearchIndexer.php

```diff
@@ -70,13 +70,35 @@ public function customIndexer(&$indexerConfig, &$indexerObject)
             $abstract = strip_tags($originalObject->getKeSearchAbstract($index));
             $content = strip_tags($originalObject->getKeSearchContent($index));
             $fullContent = $title . "\n" . $abstract . "\n" . $content;
+            $tags = [];
+            foreach ($originalObject->getCategories() as $category) {
```

Comment:
Please take care, that not every Event model use categorization. Please check with is_callable and $category with instanceof Category.